docs/glossary.rst

```diff
@@ -64,3 +64,11 @@ Glossary
     text
         This type corresponds to ``unicode`` on Python 2 and ``str`` on Python
         3.  This is equivalent to ``six.text_type``.
+
+    nonce
+        A nonce is a **n**\ umber used **once**. Nonces are used in many
```

Comment:
Looks like we need to add "Nonces" (with that capitalization) to the docs/spelling_wordlist.txt so the spellchecker understands that it is a word.